README.rst

```diff
@@ -19,8 +19,9 @@ OpenFermion is an open source effort for compiling and analyzing quantum algorit
 Getting started
 ---------------
 
-Installing OpenFermion requires pip. Make sure that you are using an up-to-date version of it.
-To install the latest development version of OpenFermion, clone `this <http://github.com/quantumlib/OpenFermion>`__ git repo,
+Installing OpenFermion requires pip. Make sure that you are using an up-to-date
+version of it. To install the latest development version of OpenFermion,
+clone `this <http://github.com/quantumlib/OpenFermion>`__ git repo,
```

Comment:
Doesn't look like anything actually changed here.

Reply:
I see, it appears that a review is required from somebody with merge rights. I wish I could set those things separately but I guess not.